nix/cardano-services/deployments/backend-deployment.yaml

```diff
@@ -0,0 +1,108 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: backend
+    network: preview
+    release: dev-preview-cardanojs
```

Comment:
I've done a bit of research about well known labels and otherwise best practices and it appears to me that the adoption of best practices requires a revision of our current set up that is more than just move then into `commonLabels`. For example, `release` is not a well-known label and also doesn't really have much of a semantic meaning (what does "`release: dev-preview-cardanojs`" even refer to?). `network` on the other hand should probably be `cardano.iog.io/network` as labels and annotations appear to always refer to fully qualified names rather than shorthands.

I've backlogged this important work as (internal ticket) [LW-7969](https://input-output.atlassian.net/browse/LW-7969).